exec/java-exec/src/main/java/org/apache/drill/exec/physical/impl/join/JoinUtils.java

```diff
@@ -65,6 +68,11 @@
   }
   private static final org.slf4j.Logger logger = org.slf4j.LoggerFactory.getLogger(JoinUtils.class);
 
+  public static final String FAILED_TO_PLAN_CARTESIAN_JOIN = String.format(
+      "This query cannot be planned possibly due to either a cartesian join or an inequality join. " +
```

Comment:
The descriptive error message is good.  However, since it is close to 200 chars..can you pls add a newline after the first statement (In the JIRA you do show the message split up on 2 lines) ?